core/src/main/java/tachyon/master/MasterSource.java

```diff
@@ -0,0 +1,180 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.master;
+
+import java.io.IOException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.codahale.metrics.Counter;
+import com.codahale.metrics.Gauge;
+import com.codahale.metrics.MetricRegistry;
+
+import tachyon.Constants;
+import tachyon.metrics.source.Source;
+
+/**
+ * A MasterSource collects a Master's internal state.
+ */
+public class MasterSource implements Source {
+  private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
+  private static final String MASTER_SOURCE_NAME = "master";
+  private final MetricRegistry mMetricRegistry = new MetricRegistry();
+  private final Counter mFilesCreated = mMetricRegistry.counter(MetricRegistry
+      .name("FilesCreated"));
+  private final Counter mCreateFileOps = mMetricRegistry.counter(MetricRegistry
+      .name("CreateFileOps"));
+  private final Counter mFilesDeleted = mMetricRegistry.counter(MetricRegistry
+      .name("FilesDeleted"));
+  private final Counter mDeleteFileOps = mMetricRegistry.counter(MetricRegistry
+      .name("DeleteFileOps"));
+  private final Counter mFilesRenamed = mMetricRegistry.counter(MetricRegistry
+      .name("FilesRenamed"));
+  private final Counter mFilesCheckpointed = mMetricRegistry.counter(MetricRegistry
+      .name("FilesCheckpointed"));
+  private final Counter mGetFileStatusOps = mMetricRegistry.counter(MetricRegistry
+      .name("GetFileStatusOps"));
```

Comment:
Could these declarations be changed to have a new line after the `=`? I think that's the style we've been moving toward.